src/pages/LoginPage.tsx

```diff
@@ -1,20 +1,60 @@
-import { Button } from "cosmian_ui";
+import { Row, Col } from "antd";
+import logo from "../assets/cosmian_logo.svg";
 
 type LoginPageProps = {
   loginWithRedirect: () => void;
+  signUpWithRedirect: () => void;
 };
-const LoginPage: React.FC<LoginPageProps> = ({ loginWithRedirect }) => {
+const LoginPage: React.FC<LoginPageProps> = ({ loginWithRedirect, signUpWithRedirect }) => {
   return (
     <main className="login-page">
-      <div className="background">
-        <div className="text-content">
-          <p className="title">Cosmian client-side encryption</p>
-          <p>Interactive tutorial and developer code examples</p>
-          <Button onClick={loginWithRedirect}>Create an account or login</Button>
-        </div>
+      <header>
+        <img loading="lazy" decoding="async" src={logo} alt="Cosmian logo white" title="Cosmian Logo" />
+      </header>
+      <div className="text-content">
+        <p className="title">
+          Cosmian
+          <br />
+          client-side encryption
+        </p>
+        <p className="subtitle">
+          Interactive tutorial and
+          <br /> developer code examples
+        </p>
+        <Row className="buttons" align={"middle"} justify={"center"}>
+          <Col>
+            <button onClick={signUpWithRedirect}>
+              SIGN UP <ButtonsIcon className="arrow" />
+            </button>
+          </Col>
+          <Col offset={1}>
+            <button onClick={loginWithRedirect}>
+              LOG IN <ButtonsIcon className="arrow" />
+            </button>
+          </Col>
+        </Row>
```

Comment:
I don't think we need thos antd elements - as we are not building a table.
Please use simple h1 and div tags, and write a simple CSS with flexbox.
If you need help with flexbox, I'll come.

Reply:
I changed it with flexbox & used the gap property to don't lte the elements touche